cmd/vm.go

```
@@ -0,0 +1,248 @@
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+	"vmwfusion/internal/types"
+	"vmwfusion/internal/vmware"
+)
+
+var (
+	vmClient = vmware.NewClient()
+	
+	// VM command flags
+	vmName     string
```

golangci-lint (1.64.8)

var `vmName` is unused

(unused)